### ISSUE TITLE
Screenshot error in stories with popups/modals

### ISSUE THREAD

**Issue by varya** [wKich/creevey#150]
I get an error for this story https://component-library-storybook.dc2-dev.elisa.fi/?path=/story/components-modal--size-s

![2021-05-20_16-57-05](https://user-images.githubusercontent.com/27215/118991459-70905800-b98c-11eb-8c9b-0fde27c4dfc1.png)

The reason to it must be that `div#root` is 0 pixels of height. But that might be the case with all the modals/popups.

**Comment by wKich** [comment]
Yep. I know about this issue. I'm going to change the default algorithm of resolving a `captureElement`, so it will work the same as in Chromatic. For now, you can explicitly set `captureElement` to `null` or `body` for those stories.
